tests/lib/rules/valid-jsdoc.js

```diff
@@ -287,6 +287,22 @@ eslintTester.addRuleTest("lib/rules/valid-jsdoc", {
                 message: "Unexpected @returns tag; function has no return statement.",
                 type: "Block"
             }]
+        },
+        {
+            code: "/**\n * Some text\n *\n * @param arg [optional] text description\n */\nfunction testFunction(arg) {}",
```

Comment:
This function shouldn't have a named argument to match the error case reported in https://github.com/eslint/eslint/issues/2270#issuecomment-91457359